src/frontend/src/components/WorkPackageStageChip.tsx

```diff
@@ -41,7 +41,7 @@ const WorkPackageStageChip: React.FC<WorkPackageStageChipProps> = ({ stage }) =>
         variant="outlined"
         sx={{
           fontSize: 14,
-          mr: '5px',
+          mr: '0px',
```

Comment:
i think this should be 5px still because if you go to like http://localhost:3000/projects/1.1.0 the two chips are touching in the work package section